docker/Dockerfile.python

```diff
@@ -38,7 +38,7 @@ RUN wget --quiet https://repo.anaconda.com/miniconda/Miniconda3-py38_4.10.3-Linu
 
 # install cuML
 
-RUN conda install -y -c rapidsai -c nvidia -c conda-forge python=3.9 cuda-toolkit=11.5 cuml=$CUML_VERSION \
+RUN conda install -y -c rapidsai -c conda-forge -c nvidia python=3.9 cudatoolkit=11.5 cuml=$CUML_VERSION \
```

Comment:
Does "cudatoolkit" work the same as "cuda-toolkit"?

Reply:
I don't think so.  cudatoolkit is what the rapids selector has shown.